### ISSUE TITLE
Discussion: Closed discussion has an add argument button 

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#1590]
**What is the problem?**
_Tested in Google Chrome in incognito mode_

When a discussion closed and there are no discussions, this is the view (screen recording). The button looks and functions as a button but does nothing.

https://github.com/user-attachments/assets/936d597f-3bd6-45b9-8160-a405b85c348f

**Why is this a problem?**
Confusing UX

**Possible solution.**
@BeccaMelhuish did we discuss this at any point?

**Comment by BeccaMelhuish** [comment]
Good catch @ssin1901 thank you! I guess hopefully it wouldn't happen often, but we do need some new texts for if it does. 

Will tag @kevincrepin for input, but I'd suggest this: 

- **Discussion now closed** - for the heading I marked yellow
- Greyed out 'Add argument' button
- Removal of the last line I marked red

![image](https://github.com/user-attachments/assets/1dc22d50-6485-4318-901a-dabe0d6eefd9)

I feel it might also be good to indicate that the reason they can't see the arguments is because there were none added, not because we've hidden it now it's closed. 

But I'm not sure how to do that smoothly ('Discussion ended, no contributions' could be the heading, but not sure that works well, and adding a replacement last line could work but takes a lot of reading). Maybe just this is enough, 0 contributions:  

![image](https://github.com/user-attachments/assets/5290ae3f-7549-405b-a3af-eda34c924806)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 seems like a 'small' and 'soon' one to me, so I labelled and moved as such

**Comment by kevincrepin** [comment]
We actually had a state for that :)

![Screenshot 2024-09-27 at 15 30 35](https://github.com/user-attachments/assets/67d303a5-b883-47e0-8914-b9909519c573)

I don't think that button should be there at all anymore, not just disabled.

**Comment by BeccaMelhuish** [comment]
Oh nice, that's much better! 

Only suggestion would be that we don't say it's currently in voting, as it may not always be the case. So perhaps this instead: 

**No discussion took place**
Nobody posted any arguments and the discussion has now been closed.
